README.md

```diff
@@ -226,6 +226,46 @@ module.exports = [
 ];
 ```
 
+It is difficult to reach 100% code coverage. Sometimes you just want to turn it off, like when you have an assertion for logical errors:
```

Comment:
Mind removing the part about being difficult, maybe rephrase to something like:

"Sometimes you want to disable code coverage for specific lines, below is an example demonstrating this:"